oca_dependencies.txt

```diff
@@ -1,3 +1,4 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
 reporting-engine
 server-ux
+web-api
```

Comment:
no need to touch this file but... yeah... maybe somebody is still using it.
Although I'm not sure this is the right place for this module (the repo I mean). Maybe "PIM" is more appropriate even if not directly bound.

Reply:
I don't know to be honest. Maybe a new repo like `web-api-contrib` (similar to `mis-builder-contrib`) would be better... 

@sbidoul @lmignon @etobella any opinion?

Reply:
I don't really have an opinion, as I don't see the immediate benefit of such a module. In my mind, exposing an API means exposing a contract and the system must at all times ensure that the data is validated against the contract while being able to provide documentation detailing the terms of the contract. 
Apart from this comment, the `web-api` repo would probably be more appropriate, or a new `web-api-contrib` repo.

Reply:
I'm proposing the new repo here https://github.com/OCA/repo-maintainer-conf/pull/27